docs/reference/migration/migrate_5_0/aggregations.asciidoc

```diff
@@ -1,13 +0,0 @@
-[[breaking_50_aggregations_changes]]
-=== Aggregation changes
-
-==== Significant terms on numeric fields
-
-Numeric fields have been refactored to use a different data structure that
-performs better for range queries. However, since this data structure does
-not record document frequencies, numeric fields can no longer be used for
-significant terms aggregations. It is recommended to use <<keyword,`keyword`>>
```

Comment:
Is it worth adding a note about potential performance degradation with numerics and a recommendation to reindex as keyword if this is an issue?